src/server/pps/cmds/cmds.go

```diff
@@ -729,11 +730,14 @@ All jobs created by a pipeline will create commits in the pipeline's output repo
 			if pipeline != "" {
 				request.Pipeline = pachdclient.NewPipeline(pipeline)
 			}
-			response, err := client.PpsAPIClient.ListPipeline(client.Ctx(), request)
+			lpClient, err := client.PpsAPIClient.ListPipeline(client.Ctx(), request)
+			if err != nil {
+				return grpcutil.ScrubGRPC(err)
```

Comment:
We probably need to scrub the other call sites as well.

Reply:
This is just for errors that go to the user right? I'm looking at what this does, and you would never want to call it for any error you wanted to inspect.